Update contributor-day.md

nagpai opened the pull request:
Minor backtick comment markdown formatting added for a few text. 

Change found while reviewing issue #48

nagpai committed:
Update contributor-day.md

Minor backtick comment markdown formatting added for a few text.

SirLouen commented on contributor-day.md:
I've been reviewing in other syncs, and we should remove all traces of HTML.

nagpai replied:
What do we replace it with? How do we indicate the note status? Use GH markdown?  @SirLouen 

> [!NOTE]  
> Show note 

```
> [!NOTE]  
> Show note 
```

SirLouen replied:
@nagpai 

I think we are going to leave the DIV, I want to see how its going to be parsed and outputted. 
I need to set up a local environment first before doing the sync to see how it goes.
The GH markdown won't work

nagpai replied:
Cool. So I am not changing anything for now.

SirLouen replied:
@nagpai I've tested it, and GFM is not working.
HTML is working, but anything inside the HTML elements won't get anything MD, they must be full HTML, for example, links should be `<a>`. 
With this in mind check the two HTML objects you have in this document and modify the PR to adapt to this.

nagpai replied:
Thanks for testing, @SirLouen . I have changed the links within the two <div> s to HTML. I hope this will work as expected. [2a09597](https://github.com/WordPress/test-handbook/pull/66/commits/2a09597cfe436a7823042956bd814ba4a7db2f50)

SirLouen commented on contributor-day.md:
Same here.

nagpai replied:
Leaving the div as it is for now, per last comment.

nagpai replied:
Changes done as per earlier comment - [2a09597](https://github.com/WordPress/test-handbook/pull/66/commits/2a09597cfe436a7823042956bd814ba4a7db2f50)

Successfulsebunya commented on contributor-day.md:
these "` " can not be parsed correctly in the .md as they represent a code block, can we use single quotes instead ( ' ' )

Replace [`wordpress-develop` repository] with ['wordpress-develop' repository]

nagpai replied:
Removed all backticks here and added single quotes for `wordpress-develop`.

SirLouen replied:
Why they cannot be parsed? 
I need to do some testing on this div blocks first to see how they are going to be parsed in a local scenario. Maybe we will need to either remove these divs, or add a PR to the parser to add the GH markdown with the special tags.

Successfulsebunya replied:
we had suggested to remove all HTML code and merge everything into .md

nagpai committed:
Remove backticks

Address PR review - Backticks need to be removed since they do not get parsed while syncing with the live docs.

nagpai committed:
Merge branch 'WordPress:trunk' into patch-1

Successfulsebunya reviewed (approved):
All looks good. Let us wait for @SirLouen 's comments tests before merging

SirLouen reviewed (changes requested):
Check for HTML changes I have commented

nagpai committed:
Merge branch 'WordPress:trunk' into patch-1

nagpai committed:
Fix link formatting in contributor-day.md tips

Updated links within tips in contributor-day.md to use HTML anchor tags for better formatting.
